karma.conf.js

```diff
@@ -101,6 +89,9 @@ module.exports = function (config) {
             'src/components/mainMenu/mainMenu.js',
             'src/components/setting/setting.js',
             'src/components/votesPreview/index.js',
+            'src/components/register/index.js', 
```

Comment:
This file has some tests added below. Does it still need to be here?

Reply:
Yes, It does not cover 80%. As far as I remember @reyraa covered it in some other ticket. @reyraa could you please confirm.